src/mongocxx/test/database.cpp

```diff
@@ -232,8 +232,7 @@ TEST_CASE("A database", "[database]") {
 
     SECTION("may create a collection") {
         MOCK_COLLECTION
-        // dummy_collection is the name the mocked collection_get_name returns
-        stdx::string_view collection_name{"dummy_collection"};
+        std::string collection_name{"dummy_collection"};
```

Comment:
I think if you made the comparator operators I suggested above, you could just say:

auto collection_name = "dummy_collection";